project-base/storefront/urql/cache/updates.ts

```diff
@@ -85,6 +85,9 @@ export const cacheUpdates: UpdatesConfig = {
         CreateOrder(_result: TypeCreateOrderMutation, _args: TypeCreateOrderMutationVariables, cache) {
             invalidateFields(cache, ['currentCustomerUser']);
         },
+        CreateDeliveryAddress(_result: TypeCreateOrderMutation, _args: TypeCreateOrderMutationVariables, cache) {
+            invalidateFields(cache, ['currentCustomerUser']);
+        },
```

Comment:
Just a question, if you edit or remove, everything works correctly?

Reply:
yeah, it works great.